docs/components/qa-service/llm-response.md

```diff
@@ -28,3 +28,42 @@ sequenceDiagram
   Semantic Search->>User: Return JSON of LLM response and K items
 
 ```
+
+## Optional components
+
+### Align Score
+
+In the Process Flow above, _Step 12: Check if answer is consistent with content_ can
+be done using [AlignScore](https://github.com/yuh-zha/AlignScore).
+
+``` mermaid
+sequenceDiagram
+  autonumber
+  User->>Semantic Search: "Aaj ka mausum kaisa hai?"
+  Semantic Search->>LLM: Identify Language
+  LLM->>Semantic Search: <Language>
+  Semantic Search->>LLM: Translate Text
+  LLM->>Semantic Search: <Translated Text>
+  Semantic Search->>LLM: Paraphrase Question
+  LLM->>Semantic Search: <Paraphrased Question>
+  Semantic Search->>Vector Db: Find K most similar items in Db
+  Vector Db->>Semantic Search: Return K items with similarity score
+  Semantic Search->>LLM: Construct response to question with content
+  LLM->>Semantic Search: <LLM Answer>
+  Semantic Search->>AlignScore: Check if answer is consistent with content
+  AlignScore->>Semantic Search: <Consistency score>
+  Semantic Search->>User: Return JSON of LLM response and K items
+
+```
+
+#### Enable Align Score
+
+To enable using AlignScore for checking content consistency, set `ALIGN_SCORE_METHOD`
+config variable to `AlignScore`. See [Configuring AAQ](../../deployment/config-options.md)
+for ways to set this.
+
+#### Setup infrastructure
+
+To use AlignScore, AAQ needs access to the AlignScore service. This can be deployed
+using a docker container. See [AlignScore](../../other-components/align-score/index.md) under
+__Other Components__ for how to do this.
```

Comment:
Also need to set the docker compose flag and `ALIGN_SCORE_API` right? Maybe we can just link to the `other-components/align-score/deployment.md` file here without further explanation?

Reply:
Thanks for this Yeah makes more sense to just link it to the the align-score docs